drivers/block-nbd.c

```diff
@@ -615,7 +624,7 @@ tdnbd_nbd_negotiate_old(struct tdnbd_data *prv, td_driver_t *driver)
 	 *
```

Comment:
Should we have a similar comment in `tdnbd_nbd_negotiate_new` below?

Reply:
Perhaps, and we would have if there had been a comment there already that could be modified.

I'll go and make one...